dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/PortRangeBean.scala

```diff
@@ -0,0 +1,8 @@
+package com.lightbend.lagom.maven
+
+import scala.beans.BeanProperty
+
+class PortRangeBean {
```

Comment:
It's a Java bean that gets instantiated and populated by maven when configuring mojos.  It will only be accessed from one thread.